src/redux/createNetworkMiddleware.js

```diff
@@ -114,7 +124,14 @@ function createNetworkMiddleware({
     }
 
     const isBackOnline = didComeBackOnline(action, isConnected);
-    if (isBackOnline) {
+    const hasQueueBeenResumed = didQueueResume(action, isQueuePaused);
+
+    const shouldDequeue =
```

Comment:
What about this regrouping, since `isBackOnline` by itself was already a valid condition?

```js
    const shouldDequeue =
      (isBackOnline || (isConnected && hasQueueBeenResumed)) &&
      shouldDequeueSelector(getState());
```

Reply:
Good idea, I'll implement this :+1: